driver/flags_table.c

```diff
@@ -111,6 +111,13 @@ const struct ppm_name_value clone_flags[] = {
 	{"NAME_CHANGED", PPM_CL_NAME_CHANGED},
 	{"CLOSED", PPM_CL_CLOSED},
 	{"CLONE_NEWUSER", PPM_CL_CLONE_NEWUSER},
+	{"CLONE_CHILD_CLEARTID", PPM_CL_CLONE_CHILD_CLEARTID},
+	{"CLONE_CHILD_SETTID", PPM_CL_CLONE_CHILD_SETTID},
+	{"SIGCHLD", PPM_CL_SIGCHLD},
```

Comment:
I have read again the `clone` documentation and it seems that the `termination signal` part is not mandatory, so the solution that I mentioned before may give false data if it is not specified.
What do you think?